wavenet/audio_reader.py

```diff
@@ -8,6 +9,27 @@
 import tensorflow as tf
 
 
+def get_category_cardinality(files):
+    id_reg_expression = re.compile(r'p([0-9]+)_([0-9]+)\.wav')
+    min_id = None
+    max_id = None
+    for filename in files:
+        matches = id_reg_expression.findall(filename)[0]
+        id, recording_id = [int(id_) for id_ in matches]
+        if id < min_id or min_id is None:
```

Comment:
Testing with python 3.4.2 this should be `if min_id is None or id < min_id:` otherwise it will throw `TypeError: unorderable types: int() < NoneType()`